quick_sort/quickSort.js

```diff
@@ -0,0 +1,78 @@
+/*
+*	Implementation of quicksort algorithm
+*	Time complexity: O(nlogn)
+* 	Space complexity: O(n)
+*	n is the size of array
+*/
+function median (arr, i, j, k) {
+	/*
+	:param arr: array of elements
+	:param i: index of first element
+	:param j: index of last element
+	:param k: index of middle element
+	:return: return median of values at indices i, j and k
+	*/
+	if (arr[i] > arr[j]) {
+		if (arr[k] > arr[j]) {
```

Comment:
The conditions here look wrong. (also in python implementation) as there can be a case when arr[k]>arr[i]>arr[j], in this case median is supposed to be `arr[i]` but we get k instead.